src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+/*        String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        ArrayList<Task> lists = new ArrayList<Task>();
```

Comment:
"lists" as a variable name should be avoided, the variable should be representing a collection of objects like "tasks"